server/server.go

```diff
@@ -5,13 +5,15 @@ import (
 	"crypto/tls"
 	"fmt"
 	"io/fs"
+	"io/ioutil"
```

Comment:
This package has been deprecated in Go 1.16 and possibly is subject to removal in a future release.

Reply:
Thank you! I did not know  package is deprecated. Switched to `os.ReadFile` as you suggested